core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala

```diff
@@ -513,7 +533,8 @@ class ReassignPartitionsCommand(zkUtils: ZkUtils,
   private[admin] def assignThrottledReplicas(existingPartitionAssignment: Map[TopicAndPartition, Seq[Int]],
                                              proposedPartitionAssignment: Map[TopicAndPartition, Seq[Int]],
                                              admin: AdminUtilities = AdminUtils): Unit = {
-    for (topic <- proposedPartitionAssignment.keySet.map(_.topic).toSeq) {
+    val topics = proposedPartitionAssignment.keySet.map(_.topic).toSeq.distinct
```

Comment:
Is there specific benefit to make this change? If not, given that different people may have different code style, it may be better to keep the existing code to avoid the back and force in style.